docs/installation/production-rack/aws.md

```diff
@@ -54,6 +54,7 @@ The following environment variables are required:
 | **node_disk**            | **20**                 | Node disk size in GB                                                                                           |
 | **node_type**            | **t3.small**           | Node instance type. You can also pass a comma separated list of instance types                                 |
 | **private**              | **true**               | Put nodes in private subnets behind NAT gateways                                                               |
+| **proxy_protocol**       | **true**               | Enable proxy protocol. With this parameter set, the client source ip will be available in the request header `x-forwarded-for` key. **Requires 5 - 10 minutes downtime**.          |
```

Comment:
Should this be `false`?

Reply:
Yep, correct, default should be false to keep backward compatibility. Thanks for spotting that